server/db/db.js

```diff
@@ -153,6 +174,7 @@ const setup = () =>
   .then(createQuestionsScriptsTable)
   .then(createCampaignsTable)
   .then(createUsersTable)
+  .then(createCallsTable)
```

Comment:
Can you add this to the teardown as well?

Reply:
Yep, I put it before campaigns since it has a campaign_id.